ext/timers/mod.js

```diff
@@ -96,27 +98,44 @@ function setTimeout(callback, timeout = 0, ...args) {
  */
 function setInterval(callback, interval = 0, ...args) {
 	if (typeof callback !== "function") {
-		throw new UnsupportedSetTimerCodeError("Interval");
+		throw new UnsupportedSetTimerCodeError("Timeout");
```

Comment:
this actually should be `Interval`, it's so that it throws error with message "set**Interval** doesn't support...."

Reply:
Whoops, copy-paste failure